Assets/AWSIM/Scripts/GeoCoordinate/GeoCoordinate.cs

```diff
@@ -0,0 +1,24 @@
+namespace AWSIM.Geographic
+{
+[System.Serializable]
+public class GeoCoordinate
+{
+  public double Latitude;
+  public double Longitude;
+  public double Altitude;
```

Comment:
Is it immutable? If it's immutable, I think it should be a getter property.

Reply:
That is certainly true.
I'll fix it to use a getter property.